int8 INDIVIDUALS=1
int8 CLASSES=2
int8 DATA_PROPERTIES=4
int8 OBJECT_PROPERTIES=8
int8 LITERAL=16

int8 source
string[] values_str
int64[] values_int
---
int8 source
string[] values_str
int64[] values_int